### DISCUSSION TITLE
Idee aus unserer Vermesserecke: Punkt zeichnen (aus aktueller Position)

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#87] [Ideas]
Die "aktuelle Position" könnte in das Kontextmenü aufgenommen werden, damit relativ rasch ein Punkt auf die aktuelle Position gesetzt werden kann, mit allen Unwägbarkeiten, die sich ergeben.  Derzeit ist es gar nicht so einfach, auf/neben "Mein Standort" einen Punkt zu setzen...

**Comment by gis-eni** [comment]
Auch bei uns gab es schon die Idee mit WebGIS zu vermessen. Es gibt spezielle Antennen auf die man sich auch über einen Mobiles Geräte (Hände/Tablet) verbinden kann und Genauigkeiten <10cm erreichen kann.

Von der Bedienung schaut das für den Anwender dann so aus, dass es beim Zeichnen auf mobilen Geräten noch einen weiteren Button "GPS" gibt:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/12de57f3-bde7-463a-a585-bdac07da18d9)

Ist dieser Button in der "Ruhestellung" (wie oben) kann der Anwender wie gewohnt in der Karte zeichnen. Wird die "GPS" Bubble aber irgendwohin in die Karte gezogen, folgt der Zeichencursor dem GPS Signal:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/16f7dae0-5423-4b9a-b947-35ec35be47f2)

Wenn eine gewünschte Genauigkeit gegeben ist wird die Bubble Grün und der Anwender kann durch einen Klick auf die Bubble einen Punkt auf die aktuelle GPS Position setzten.
So kann der Anwender ein Linie oder Fläche abgehen und Punkte setzen.

``` javascript

if (/*mapUrlName === "Basiskarte und Kataster"*/ true) {
    webgis.currentPosition = webgis.currentPosition_watch;

    webgis.currentPosition.minAcc = 500000;   // [m]
    webgis.currentPosition.maxAgeSeconds = 0.1;  // [s]
    webgis.currentPosition.useWithSketchTool = true; 

    webgis.continuousPosition.useTransformationService = false;
    webgis.continuousPosition.transformationSpatialValidity = 20;

    //webgis.continuousPosition.helmert2d = {
    //    srs: 31256,
    //    Cx:  0.600,
    //    Cy: -0.234,
    //    Rx: -67946.151,
    //    Ry: 215079.498,
    //    r: 399.9992 * Math.PI / 200,
    //    scale: 1 + (-6.576 * 1e-6)
    //};
}

```
